src/index.js

```diff
@@ -18,7 +28,30 @@ function lessLoader(source) {
     );
   }
 
-  processResult(loaderContext, render(source, options));
+  const lessOptions = {
```

Comment:
Let's move it to `getLessOptions` (example from sass-loader https://github.com/webpack-contrib/sass-loader/blob/master/src/getSassOptions.js)